### PR TITLE
Added missing land pattern reference label and position indicator 

### PR DESCRIPTION
**Pull request by m-hilgendorf**
Before:

bga-before

After:
bga-after

### CODE REVIEW

File: utils/land-patterns.stanza [hunk 1]
```diff
@@ -2018,6 +2018,22 @@ public defn make-bga-landpattern (num-rows:Int, ; the number of rows of package
...
+    ;-------------------------------------------------------------
+    ; Compute the orientation marker
+    val silk-solder-mask-clearance = clearance(current-rules(), MinSilkSolderMaskSpace)
+    val solder-mask-registration = clearance(current-rules(), SolderMaskRegistration)
+    val line-width = clearance(current-rules(), MinSilkscreenWidth)
+    val pad-loc = center(pad-locs[0])
+    val pol-x = x(pad-loc) - ball-diameter * 0.5 - solder-mask-registration
+    val pol-y = y(pad-loc) + ball-diameter * 0.5 + solder-mask-registration
+    val line-length = 3.0 * line-width
+    val dist  = solder-mask-registration + silk-solder-mask-clearance + line-width * 0.5 + 0.01
```
Comment (ariel-hi) [inline utils/land-patterns.stanza:2030]: I'm curious about the 3.0 and 0.01 constants.
Is the length being 3 times the width a standard or it just looks nice that way?
Not sure why the 0.01 is useful?

Comment (m-hilgendorf) [reply utils/land-patterns.stanza:2030]: They're magic numbers from @electricfungi

Comment (ariel-hi) [reply utils/land-patterns.stanza:2030]: 🪄 💫 ⭐

**Review by ariel-hi — APPROVED** [review]

File: utils/land-patterns.stanza [hunk 2]
```diff
@@ -2001,7 +2001,7 @@ public defn make-bga-landpattern (num-rows:Int, ; the number of rows of package
           [r, c]
     
     ; Calculate pad locations
-    val pad-locs  = locs(modifier, num-rows, num-cols, pitch)
+    val pad-locs = to-tuple(locs(modifier, num-rows, num-cols, pitch))
```
Comment (bpellenard) [inline utils/land-patterns.stanza:2004]: `to-tuple $` to minimize the number of parenthesis

**Pull request merged**